Dockerfile

```diff
@@ -0,0 +1,83 @@
+# Base Image
+FROM ncbihackathon/ncbihackathonbase:latest
+
+# Metadata
+LABEL base.image="ncbihackathonbase:latest"
+LABEL version="1"
+LABEL software="NCBI Hackathon Image for NGS analysis"
+LABEL software.version="0.0.1"
+LABEL description="NCBI Hackathons Image for NGS analysis"
+LABEL website="https://github.com/NCBI-Hackathons/HackathonDockerImages/Docker/ngs"
+LABEL documentation="https://github.com/NCBI-Hackathons/HackathonDockerImages/Docker/ngs"
+LABEL license="https://github.com/NCBI-Hackathons/HackathonDockerImages/LICENSE"
+LABEL tags="NCBI, Hackathon, Bioconductor"
+
+# Maintainer
+MAINTAINER Roberto Vera Alvarez <r78v10a07@gmail.com>
+
+USER biodocker
+
+RUN git clone --recursive https://github.com/NCBI-Hackathons/NanoporeMapper.git
```

Comment:
The URL for the repo has changed to https://github.com/NCBI-Hackathons/STREAMclean.git (the old URL works as GitHub handles redirects when repos are renamed, but would probably be nicer to have a clean setup)